curriculum/templates/curriculum/lessonplan_detail.html

```diff
@@ -14,142 +14,132 @@
+                    <h6 class="h6 font-weight-bold my-3">Leave a review</h6>
+                    {% if user.is_authenticated %}
+                        <div class="col-md-8 p-0">
+                            {% crispy form form.helper %}
+                        </div>
+                    {% else %}
+                        <p>You must be logged in to leave a review.</p>
+                    {% endif %}
+                </div>
+            {% else %}
+            <h6 class="h6 font-weight-bold my-3">Reviews have been disabled.</h6>
+            {% endif %}
+            <div class="list-group-item">
+                <h6 class="h6 font-weight-bold my-3">Reviews</h6>
                 {% if lessonplan.lessonfeedback_set.all %}
-                    <div class="list-group-item d-flex flex-wrap align-items-center">
-                        <p class="my-0">Average Rating: {{ average_rating|floatformat:2 }}</p>
-                        <p class="flex-fill my-0 text-center stars text-primary" data-average-rating="{{ average_rating }}" >
-                            <i class="fa-star" aria-hidden="true"></i>
-                            <i class="fa-star" aria-hidden="true"></i>
-                            <i class="fa-star" aria-hidden="true"></i>
-                            <i class="fa-star" aria-hidden="true"></i>
-                            <i class="fa-star" aria-hidden="true"></i>
-                        </p>
-                        <a class="flex-fill btn btn-outline-primary my-0" data-toggle="collapse" href="#feedback" role="button" aria-expanded="false" aria-controls="feedback">
-                            See Detailed Feedback
-                        </a>
-                    </div>
-                    <div class="collapse" id="feedback">
+                    <div id="feedback">
                         {% for fb in lessonplan.lessonfeedback_set.all %}
-                        <div class="list-group-item">
-                            <div class="d-flex">
-                                <p class="text-muted flex-fill">Submitted by {{ fb.author.first_name }} from {{ fb.author.profile.school }}.</p>
-                                <p class="text-right text-primary" data-average-rating="{{ fb.overall_rating }}" >
-                                    <i class="fa-star" aria-hidden="true"></i>
-                                    <i class="fa-star" aria-hidden="true"></i>
-                                    <i class="fa-star" aria-hidden="true"></i>
-                                    <i class="fa-star" aria-hidden="true"></i>
-                                    <i class="fa-star" aria-hidden="true"></i>
-                                </p>
-                            </div>
+                        <div class="col-lg-8 list-group-item px-0">
+                            <p class="mb-4 stars text-primary" data-average-rating="{{ fb.overall_rating }}" >
+                                <i class="fa-star" aria-hidden="true"></i>
```

Comment:
I think the style might end up applied anyways from my PR, so might be unnecessary - but thoughts on changing the starts color to the gray background to make it consistent with the Find&Share Curricula?

Reply:
Awesome catch, they're gray in zeplin as well! Pushed the changes.